twittermap/public/javascripts/common/services.js

```diff
@@ -2,6 +2,7 @@ angular.module('cloudberry.common', [])
   .factory('cloudberryConfig', function(){
     return {
       ws: config.wsURL,
+      sentimentAnalysisEnabled: true,
```

Comment:
can this parameter be set as the `ws` such that we only need to change the `application.conf` file without touching the code?